src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/Browser/WasmBrowserTestRunner.cs

```diff
@@ -163,6 +163,8 @@ private async Task RunConsoleMessagesPump(WebSocket socket, TaskCompletionSource
                     if (result.EndOfMessage)
                     {
                         var line = Encoding.UTF8.GetString(mem.GetBuffer(), 0, (int)mem.Length);
+                        line += Environment.NewLine;
```

Comment:
Is this re-adding the newlines in between logger output messages? I really don't think we want this. It means every mono log entry will have them, which blows up quite quickly.

Reply:
This only applies to the browser runner, it turned out that with https://github.com/dotnet/xharness/pull/314 the non-browser (v8) runner would insert blank newlines after each line in the XML.

With this change we'll only have the failed tests output in the console by default, the normal stdout goes to wasm-console.log which has no blank lines.

Reply:
The mono logs not getting the newlines is good to hear, though I still don't follow why the non-browser runner is important here. Your sample output above seems like it has more newlines than I'd prefer. Desktop runner output:
```
  ~/git/runtime/artifacts/bin/System.Reflection.Tests/net5.0-release ~/git/runtime/src/libraries/System.Reflection/tests
    Discovering: System.Reflection.Tests (method display = ClassAndMethod, method display options = None)
    Discovered:  System.Reflection.Tests (found 393 of 410 test cases)
    Starting:    System.Reflection.Tests (parallel test collections = on, max threads = 12)
      System.Reflection.Tests.TypeInfoTests.Namespace(type: typeof(System.Reflection.Tests.TI_BaseClass+PublicNestedClass1[]), expected: "System.Reflection.Tests") [FAIL]
        Assert.Equal() Failure
        Expected: System.Reflection.Tests
        Actual:   (null)
        Stack Trace:
          /Users/ryan/git/runtime/src/libraries/System.Reflection/tests/TypeInfoTests.cs(1414,0): at System.Reflection.Tests.TypeInfoTests.Namespace(Type type, String expected)
          /Users/ryan/git/runtime/src/mono/netcore/System.Private.CoreLib/src/System/Reflection/RuntimeMethodInfo.cs(384,0): at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
    Finished:    System.Reflection.Tests
  === TEST EXECUTION SUMMARY ===
     System.Reflection.Tests  Total: 1283, Errors: 0, Failed: 1, Skipped: 0, Time: 0.941s
  ~/git/runtime/src/libraries/System.Reflection/tests
```

Reply:
The issue is basically https://github.com/dotnet/xharness/pull/314#issuecomment-699985617

I'll make it a bit more similar to the desktop runner output.

Reply:
Turns out the newest version of the Microsoft.Extensions.Logging package made improvements in that area and we can now get much closer to what we want (still no easy way to strip out the `test[0]` though).

It now looks like this:

```
  info: test[0] Discovering: System.Runtime.Handles.Tests.dll (method display = ClassAndMethod, method display options = None)
  info: test[0] Discovered:  System.Runtime.Handles.Tests.dll (found 14 of 15 test cases)
  info: test[0] Starting:    System.Runtime.Handles.Tests.dll
  fail: test[0] [FAIL] SafeWaitHandleTests.SafeWaitHandle_False
  info: test[0] Assert.False() Failure
  info: test[0] Expected: False
  info: test[0] Actual:   True
  info: test[0]    at SafeWaitHandleTests.SafeWaitHandle_False()
  info: test[0]    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
  info: test[0] Finished:    System.Runtime.Handles.Tests.dll
  info: test[0]
  info: test[0] === TEST EXECUTION SUMMARY ===
  info: test[0] Total: 14, Errors: 0, Failed: 1, Skipped: 0, Time: 0.250726s
```